fermenter-mda/src/test/java/org/technologybrewery/fermenter/mda/LanguageSpecificPluginConfigSteps.java

```diff
@@ -24,54 +28,55 @@ public class LanguageSpecificPluginConfigSteps {
     }
 
-    @When("^the project's pom\\.xml is processed$")
+    @When("the project's pom.xml is processed")
     public void the_project_s_pom_xml_is_processed() throws Throwable {
         generateSourcesMojo = (GenerateSourcesMojo) mojoTestCase.lookupConfiguredMojo(new File(mavenProjectBaseDir, "pom.xml"), "generate-sources");
         generateSourcesMojo.updateMojoConfigsBasedOnLanguage();
     }
 
-    @When("^the Fermenter Maven plugin configuration is validated$")
+    @When("the Fermenter Maven plugin configuration is validated")
     public void the_Fermenter_Maven_plugin_configuration_is_validated() throws Throwable {
         generateSourcesMojo.validateMojoConfigs();
     }
 
-    @Then("^Fermenter Maven plugin uses \"([^\"]*)\" as the main source root folder, \"([^\"]*)\" as the generated source root folder, and \"([^\"]*)\" as the base package$")
+    @Then("Fermenter Maven plugin uses {string} as the main source root folder, {string} as the generated source root folder, and {string} as the base package")
     public void fermenter_Maven_plugin_uses_as_the_main_source_root_folder_as_the_generated_source_root_folder_and_as_the_base_package(String mainSourceRoot, String generatedSourceRoot, String namespace) throws Throwable {
         validatePluginConfig(generateSourcesMojo.getMainSourceRoot(),
             new File(mavenProjectBaseDir, mainSourceRoot), "mainSourceRoot");
         validatePluginConfig(generateSourcesMojo.getGeneratedSourceRoot(),
             new File(mavenProjectBaseDir, generatedSourceRoot), "generatedSourceRoot");
-        Assert.assertEquals("Namespace/base package did not match the expected value", namespace,
-            generateSourcesMojo.getBasePackage());
+        assertEquals(namespace, generateSourcesMojo.getBasePackage(), "Namespace/base package did not match the expected value");
     }
 
-    @Then("^Fermenter Maven plugin uses \"([^\"]*)\" as the local metadata root folder$")
+    @Then("Fermenter Maven plugin uses {string} as the local metadata root folder")
     public void fermenter_Maven_plugin_uses_as_the_local_metadata_root_folder(String localMetadataRoot) throws Throwable {
         // Write code here that turns the phrase above into concrete actions
         validatePluginConfig(generateSourcesMojo.getLocalMetadataRoot(),
             new File(mavenProjectBaseDir, localMetadataRoot), "localMetadataRoot");
     }
 
     protected void validatePluginConfig(File actualPluginConfigFile, File expectedPluginConfigFile, String configDescription) throws IOException {
-        Assert.assertNotNull(String.format("%s was unexpectedly null", configDescription), actualPluginConfigFile);
-        Assert.assertEquals(String.format("%s path did not equal expected value", configDescription),
-            expectedPluginConfigFile.getCanonicalPath(), actualPluginConfigFile.getCanonicalPath());
+        assertNotNull(actualPluginConfigFile, String.format("%s was unexpectedly null", configDescription));
```

Comment:
I: JUnit Jupiter was kind enough to switch the passing order of the failure message from first to last.